.github/workflows/build-m1-binaries.yml

```diff
@@ -71,3 +70,52 @@ jobs:
           for pkg in dist/*; do
             aws s3 cp "$pkg" "s3://pytorch/whl/${CHANNEL}/cpu/" --acl public-read
           done
+  build_conda:
+    name: "Build TorchVision M1 conda packages"
+    runs-on: macos-m1
+    strategy:
+      matrix:
+        py_vers: [ "3.8", "3.9", "3.10" ]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install conda-build and purge previous artifacts
+        shell: arch -arch arm64 bash {0}
+        run: |
+          . ~/miniconda3/etc/profile.d/conda.sh
+          conda install -yq conda-build
+          conda build purge-all
+
+      - name: Build TorchVision M1 conda package
+        shell: arch -arch arm64 bash {0}
+        env:
+          ENV_NAME: conda-env-${{ github.run_id }}
+          PYTHON_VERSION: ${{ matrix.py_vers }}
+          CU_VERSION: cpu
+        run: |
+          . ~/miniconda3/etc/profile.d/conda.sh
+          set -ex
+          . packaging/pkg_helpers.bash
+          setup_build_version
+          setup_conda_pytorch_constraint
+          export SOURCE_ROOT_DIR=$(pwd)
+          conda build -c defaults $CONDA_CHANNEL_FLAGS --no-anaconda-upload --python "$PYTHON_VERSION" packaging/torchvision
+          mkdir -p dist
+          cp ~/miniconda3/conda-bld/osx-arm64/*.tar.bz2 dist/
```

Comment:
Can we test the conda package by running the smoke tests as we do above with the wheels?

Reply:
@datumbox isn't it part of coda build process? I.e. it should execute commands defined in `test` section, i.e.:
https://github.com/pytorch/vision/blob/d4a03fc02d0566ec97341046de58160370a35bd2/packaging/torchvision/meta.yaml#L53-L63

Reply:
It will just try to import `torchvision`, `torchvision.datasets` and `torchvision.transforms`?
We should at least check that ops are properly present like we do for the wheels

```
          conda run --cwd /tmp -p ${ENV_NAME} python3 -c "import torchvision;print(torchvision.io.read_image('${PWD}/gallery/assets/dog1.jpg').shape)"
```